config/config.yaml

```diff
@@ -76,7 +76,7 @@ exporters:
 service:
     ## Add basicauth/* if you want to use basic auth extensions
     ## Add bearertokenauth/* if you want to use bearer token auth extensions
-    extensions: [health_check, pprof, zpages]
+    extensions: [health_check, pprof, zpages, bearertokenauth/gitlab]
```

Comment:
we should restore this back to the previous commit. there's no need to commit updates to this past the last update.

Reply:
The latest commit should address this.